docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+=== `table`
+
+Configures a data table element.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`font`
+|`style`
+|Font style 
+
+Default: `{<<font_fn>>}`
+
+|`paginate`
+|`boolean`
+|Show pagination controls. If set to `false` only the first 
+page is displayed.  
+
+Default: `true`
+
+|`perPage`
+|`number`
+|Show this many rows per page 
+
+Default: `10`
+
+|`showHeader`
+|`boolean`
+|Show or hide the header row with titles for each column 
+
+Default: `true`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[tail_fn]]
+=== `tail`
+
+Gets the last N rows from the end of a `datatable`. See also <<head_fn>>
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `count`
+|`number`
+|Return this many rows from the end of the `datatable`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[timefilter_fn]]
+=== `timefilter`
+
+Creates a time filter for querying a source.
+
+*Accepts:* `filter`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`column` 
+
+Alias: `field`, `c`
+|`string`
+|The column or field to attach the filter to 
+
+Default: `"@timestamp"`
+
+|`from` 
+
+Alias: `f`, `start`
+|`string`
+|Beginning of the range, in ISO8601 or {es} `datemath` format
+
+|`to` 
+
+Alias: `t`, `end`
+|`string`
+|End of the range, in ISO8601 or {es} `datemath` format
+|===
+
+*Returns:* `filter`
+
+
+[float]
+[[timefilterControl_fn]]
+=== `timefilterControl`
+
+Configures a time filter control element.
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`column` 
+
+Alias: `field`, `c`
+|`string`
+|The column or field to attach the filter to 
+
+Default: `"@timestamp"`
+
+|`compact`
+|`boolean`
+|Show the time filter as a button that triggers a popover 
+
+Default: `true`
+
+|`filterGroup`
+|`string`
+|Group name for the filter
+|===
+
+
+*Returns:* `render`
+
+
+[float]
+[[timelion_fn]]
+=== `timelion`
+
+Uses Timelion to extract one or more time series from many sources.
+
+*Accepts:* `filter`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Aliases: `q`, `query`
+|`string`
+|A Timelion query 
+
+Default: `".es(*)"`
+
+|`interval`
+|`string`
+|Bucket interval for the time series 
+
+Default: `"auto"`
+
+|`from`
+|`string`
+|{es} `datemath` string for the start of the time range 
+
+Default: `"now-1y"`
+
+|`to`
+|`string`
+|{es} date math `string` for the end of the time range 
+
+Default: `"now"`
+
+|`timezone`
+|`string`
+|Time zone for the time range 
+
+Default: `"UTC"`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[to_fn]]
+=== `to`
+
+Explicitly casts from one type to another.
+
+*Accepts:* `any`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `type`
+|`string`
+|A known type
+|===
+
+*Returns:* Depends on your input and arguments
+
+
+[float]
+[[u_fns]]
+== U
+[[urlparam_fn]]
+=== `urlparam`
+
+Accesses URL parameters and use them in expressions. This will always return a `string`.
```

Comment:
How about:

Accesses the URL parameters, then uses the parameters in expressions. The `urlparam` function always returns a `string`.

Reply:
What about:

Retrieves the URL parameters, then uses the parameters in expressions. The `urlparam` function always returns a `string`. For example, you can retrieve the value of `"20"` from the parameter `myVar` in the URL `https://localhost:5601/app/canvas?myVar=20`.